API/BDBOAuth1SessionManager+SFAuthenticationSession.h

```diff
@@ -1,11 +1,3 @@
-//
```

Comment:
I wouldn't change third party code for lots of reasons. Mostly it's just a waste of time since they won't persist when you update the dependency.